src/DataGrid.tsx

```diff
@@ -37,7 +37,6 @@ import {
   RowExpandToggleEvent,
   RowRendererProps,
   RowsUpdateEvent,
-  ScrollPosition,
```

Comment:
This type isn't used anywhere anymore, let's delete it.

Reply:
Nice catch, I didn't know this detail.